x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/translations.tsx

```diff
@@ -192,3 +194,26 @@ export const GROUP_BY_FIELD_LICENSE_WARNING = i18n.translate(
     defaultMessage: 'Alert suppression is enabled with Platinum license or above',
   }
 );
+
+export const ENABLE_THRESHOLD_SUPPRESSION_LICENSE_WARNING = i18n.translate(
+  'xpack.securitySolution.detectionEngine.createRule.stepDefineRule.enableThresholdSuppression.licenseWarning',
+  {
+    defaultMessage: 'Alert suppression is enabled with Platinum license or above',
```

Comment:
I I discovered an upselling service that we have for security solution, which if I understood correctly suppose to be a single point for upselling (message, sections and pages). Correct me if I'm wrong @semd 

Here is an example how to register message there https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution_ess/public/upselling/register_upsellings.tsx#L104

And here the upselling integration for "alert assignments" feature https://github.com/elastic/kibana/pull/172014

Reply:
Thank you pointing out at this.
Since it touches existing custom query rule alert suppression functionality, I am going to make a separate PR after this one to refactor alert suppression upselling.

Here is a draft: https://github.com/vitaliidm/kibana/pull/10